src/main/webapp/app/exercises/shared/dashboards/tutor/abstract-assessment-dashboard.ts

```diff
@@ -3,12 +3,19 @@ import { Submission } from 'app/entities/submission.model';
 
 enum AssessmentFilter {
```

Comment:
Below you are comparing `filterOption` (a number) to this enum. Maybe explicitly assign the numbers here?
Or you could make `filterOption` as string and then merge the `AssessmentFilter` and `AssessmentTranslation` enums, which would also make the URL more readable I suppose.